client/src/layouts/Snackbar/Styles.ts

```diff
@@ -0,0 +1,7 @@
+import {styled} from '@mui/material/styles';
+import CloseIcon from '@mui/icons-material/Close';
+
+export const Icon = styled(CloseIcon)(() => ({
+    color: 'white',
+    fontSize: 16,
+}));
```

Comment:
I think we can import icon directly in Snackbar.tsx and add style inside CloseIcon.
for example:
`<CloseIcon fontSize="small" htmlColor="white" />`

Reply:
Great idea!